app/test_main.py

```diff
@@ -1,3 +1,40 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+class TestCatAndDogYears:
+    @pytest.mark.parametrize(
+        "cat_years, dog_years, expended_years",
+        [
+
+            pytest.param(
+                0, 0, [0, 0],
+                id="Should return [0, 0] when years equals zero"
+            ),
+            pytest.param(
+                14, 14, [0, 0],
+                id="Should return [0, 0] when years <= 15 years"
+            ),
+            pytest.param(
+                15, 15, [1, 1],
+                id="Should return [1, 1] when age cat equals\
```

Comment:
And another one here: `age cat` instead of `cat age`. 
By the way, since this string is enclosed in parentheses anyway, you can end string literal here and continue on the next line: 
```python
id="..."
   "..."